library/Fission/User/DID/Types.hs

```diff
@@ -0,0 +1,15 @@
+module Fission.User.DID.Types (DID (..)) where
+
+import Fission.Prelude
+
+import Data.Swagger
+import Fission.Internal.Orphanage.ByteString ()
+
+newtype DID = DID { unDID :: ByteString }
+  deriving          ( Eq
+                    , Generic
+                    , Show
+                    , ToJSON
```

Comment:
Getting this warning:
```
Both DeriveAnyClass and GeneralizedNewtypeDeriving are enabled
      Defaulting to the DeriveAnyClass strategy for instantiating ToJSON
    • In the newtype declaration for ‘DID’
```
Shouldn't cause an issue, but is there a way to suppress this warning?

Reply:
You should specify how you want these derived on the newtype (i.e. use the underlying type’s instance, or treat it as a `data`). So, likely:

```haskell
newtype DID = DID { unDID :: Text }
  deriving anyclass (Eq, Show, Generic)
  deriving newtype (ToJSON)
```